auto-complete.js

```diff
@@ -75,7 +75,7 @@
                 // escape special characters
                 search = search.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
                 var re = new RegExp("(" + search.split(' ').join('|') + ")", "gi");
-                return '<div class="autocomplete-suggestion" data-val="' + item + '" data-index="' + suggestionIndex + '">' + item.replace(re, "<b>$1</b>") + '</div>';
```

Comment:
If we are removing the bold maybe just remove this `regex` and `replace` would do it?